build.gradle

```diff
@@ -197,6 +198,35 @@ dependencyManagement {
     }
 }
 
+pitest {
+
+    targetClasses = ['uk.gov.hmcts.ccd.*']
+
+    targetTests = ['uk.gov.hmcts.ccd.auth.*',
+                   'uk.gov.hmcts.ccd.domain.model.*',
+                   'uk.gov.hmcts.ccd.domain.service.*'
+    ]
+
+    excludedClasses = ['uk.gov.hmcts.ccd.UserProfileApplication']
+
+    enableDefaultIncrementalAnalysis = true
+    historyInputLocation = ['build/reports/pitest/fastermutationtesting']
+    historyOutputLocation = ['build/reports/pitest/fastermutationtestingoutput']
+    threads = 15
+    testSourceSets = [sourceSets.test]
+    mainSourceSets = [sourceSets.main]
+    fileExtensionsToFilter.addAll('xml','json')
+    outputFormats = ['XML', 'HTML','CSV']
+    mutationThreshold = 9
+    coverageThreshold = 0
```

Comment:
coverage threshold 0?

Reply:
Thank you for your query...

I have maintained the coverage threshold and the mutation threshold to low numbers in order to get the builds to pass.

The Implementation is done and it is now up to the teams to strengthen and better their tests and push the coverage values up to meet the standards.

Hope this helps...